classes/migration/install/FailedJobsMigration.inc.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @file classes/migration/install/FailedJobsMigration.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class FailedJobsMigration
+ * @brief Describe database table structures.
+ */
+
+namespace PKP\migration\install;
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class FailedJobsMigration extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up()
+    {
+        Schema::create('failed_jobs', function (Blueprint $table) {
+            $table->bigIncrements('id');
```

Comment:
For consistency with our database design elsewhere in the system, could you call this `failed_job_id`?

Reply:
This is a copy of the [Laravel's default failed_jobs table](https://github.com/illuminate/queue/blob/7.x/Console/stubs/failed_jobs.stub). I'm afraid about changing it and crashing the framework internals.

Reply:
OK, let's leave it as `id` then, but could you note that source in the file so we know (and credit) where it came from?